src/main/java/org/jabref/gui/util/comparator/ReadStatusFieldComparator.java

```diff
@@ -0,0 +1,32 @@
+package org.jabref.gui.util.comparator;
+
+import java.util.Comparator;
+import java.util.Optional;
+
+import org.jabref.gui.specialfields.SpecialFieldValueViewModel;
+
+public class ReadStatusFieldComparator implements Comparator<Optional<SpecialFieldValueViewModel>> {
+
+    @Override
+    public int compare(Optional<SpecialFieldValueViewModel> val1, Optional<SpecialFieldValueViewModel> val2) {
+        if (val1.isPresent()) {
+            if (val2.isPresent()) {
+                int compareToRes = val1.get().getValue().compareTo(val2.get().getValue());
```

Comment:
Since you always return the same value as `compareToRes` in the end, the following code sufficies
```suggestion
               return val1.get().getValue().compareTo(val2.get().getValue());
```

Reply:
Done